Automattic/sensei
# Add progress validation command

Resolves #7079

## Proposed Changes
* Add a WP CLI command that validates the migrated progress data.
* Fixes quiz progress not migrating when multiple users have progress on the same quiz.

## Testing Instructions


1. Enable the HPPS feature `add_filter( 'sensei_feature_flag_tables_based_progress', '__return_true' );`.
2. Run the student progress migration following the testing instructions at https://github.com/Automattic/sensei/pull/7049.
3. Run the following command in your WordPress folder: `wp sensei validate progress`.
4. You should see `Success: Progress data is valid.`.
5. Try deleting a progress entry in your `wp_sensei_lms_progress` table.
6. You should see a `Course/Lesson/Quiz tables based progress not found` error.
7. Try changing the progress status in your `wp_sensei_lms_progress` table.
8. You should see a `Data mismatch between comments and tables based progress.` error.


## Pre-Merge Checklist
